api/proto/teleport/plugins/v1/plugin_service.proto

```diff
@@ -165,4 +185,10 @@ service PluginService {
   // SearchPluginStaticCredentials returns static credentials that are searched for. Only accessible by RoleAdmin and,
   // in the case of Teleport Assist, RoleProxy.
   rpc SearchPluginStaticCredentials(SearchPluginStaticCredentialsRequest) returns (SearchPluginStaticCredentialsResponse);
+
+  // NeedsCleanup will indicate whether a plugin of the given type needs cleanup before it can be created.
+  rpc NeedsCleanup(NeedsCleanupRequest) returns (NeedsCleanupResponse);
```

Comment:
Do we really need this extra RPC method? I understand this is probably for UI to determine whether a particular plugin needs a cleanup but can't we just do this only for "okta" plugin type on the frontend and be done with it? @kimlisa

Reply:
Actually, looking at the OSS PR I think I understand why this is needed - it will only return "needs cleanup" if we actually have orphaned resources from the old instance of Okta still. Nevermind then.